packages/at_secondary_server/lib/src/connection/outbound/outbound_client.dart

```diff
@@ -301,18 +310,30 @@ class OutboundClient {
   Future<String?> plookUp(String key) async {
     var result = await lookUp(key, handshake: false);
     lastUsed = DateTime.now();
+    logger.finer('plookup result of the $key: $result');
     return result;
   }
 
   void close() {
     if (outboundConnection != null) {
       outboundConnection!.close();
+      logger.finer('Outbound connection closed');
     }
   }
 
   bool isInValid() {
-    return inboundConnection.isInValid() ||
-        (outboundConnection != null && outboundConnection!.isInValid());
+    bool isInvalid = false;
+    if (inboundConnection.isInValid()) {
+      logger.severe('Outbound Connection to $toAtSign is invalid');
+      isInvalid = true;
+    }
+    if (outboundConnection != null && outboundConnection!.isInValid()) {
+      logger.severe(
+          'Inbound connection from ${inboundConnection.initiatedBy} is invalid');
+      isInvalid = true;
+    }
+    logger.severe('Invalid Connection: ${toString()}');
```

Comment:
What are you intending to log here?

Reply:
I am trying to log the details of the Outbound client, hence calling the toString() in the same class. The idea is that OutboundClient.toString() displays the atsigns were dealing with and the status of the connection, handshake, etc.